src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -38,4 +53,13 @@ class HdcService(
       )
     }
   }
+
+  data class HdcStatuses(val approvedIds: Set<Long>) {
```

Comment:
little wrapper class to provide utils to interpret HDC approval status in one place